applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -224,7 +254,33 @@ def __main():
                         const=True,
                         help="Run nrfjprog sequentially instead of in \
                         parallel.")
+    #DFU relative option
+    parser.add_argument("-m", "--min_b0n", default=False,
+                        action='store_true',
+                        help="net core bootloader use minimal size build")
+    group = parser.add_mutually_exclusive_group()
```

Comment:
Maybe it is easier to have a --mcuboot  {external, internal} argument with choices internal flash or external flash?

Reply:
Sure.
Will refactor after https://github.com/nrfconnect/sdk-nrf/pull/7542 merged.

Reply:
Change parameter to --mcuboot {external, internal} stype.